examples/dreambooth/train_dreambooth_flax.py

```diff
@@ -75,21 +73,19 @@ def parse_args():
     parser.add_argument(
         "--instance_data_dir",
         type=str,
-        default=None,
         required=True,
         help="A folder containing the training data of instance images.",
     )
     parser.add_argument(
         "--class_data_dir",
         type=str,
         default=None,
-        required=False,
         help="A folder containing the training data of class images.",
     )
     parser.add_argument(
         "--instance_prompt",
         type=str,
-        default=None,
+        required=True,
```

Comment:
This is not the same thing no? Now we would get an error if we don't set `--instance_prompt` I think, but if someone doesn't want to do validation there is no need to set `--instance_prompt` I believe